src/components/search/partials/search.html

```diff
@@ -1,4 +1,42 @@
 <form class="pull-left">
-  <input class="input-small" type="search" placeholder="{{'search_placeholder' | translate}}" ng-model="query" autocapitalize="none" autocomplete="off" autocorrect="off">
-  <i class="icon-remove-sign ng-hide" ng-click="clearInput()" ng-hide="query==''"></i>
+  <span class="ga-search-input-container">
+    <input class="input-small ga-search-input" type="search"
+    placeholder="{{'search_placeholder' | translate}}" ng-model="query"
+    autocapitalize="none" autocomplete="off" autocorrect="off"
+    ng-keyup="key($event)"
```

Comment:
The function `scope.key` doesn't modify a scope value . I suggest we don't use `ng-XXX` but jquery binding instead to avoid triggering useless digest cycle.